AUTHORS.md

```diff
@@ -11,52 +11,62 @@ contributions are subject to the project's copyright under the terms of the
 
 This file keeps track of authors contributions.
 
-## Maintainers / Steering Comittee
+## Maintainers / Steering committee
 
 ---
 
-- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
-- **Amaury Dehecq** [@adehecq](https://github.com/adehecq)
-- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github.com/erikmannerfelt)
-- **Emmanuel Dubois** [@duboise-cnes](https://github.com/duboise-cnes) <emmanuel.dubois@cnes.fr>
-- **Valentine Bellet** [@Valentine-Bellet](https://github.com/valentine-bellet) <valentine.bellet@cnes.fr>
-- **Alice de Bardonnèche-Richard** [@adebardo](https://github.com/adebardo) <alice.de-bardonneche-richard@cs-soprasteria.com>
-
+| Maintainer                       | GitHub                                     | Affiliation                    | Email                                   |
+|----------------------------------|--------------------------------------------|--------------------------------|-----------------------------------------|
+| **Romain Hugonnet**              | [@rhugonnet](https://github.com/rhugonnet) | University of Alaska Fairbanks | [📧](mailto:romain.hugonnet@gmail.com)  |
+| **Amaury Dehecq**                | [@adehecq](https://github.com/adehecq)     | Université Grenoble Alpes      | N/A                                     |
+| **Valentine Bellet**             | [@belletva](https://github.com/belletva)   | French Space Agency (CNES)     | [📧](mailto:valentine.bellet@cnes.fr)   |
+| **Alice de Bardonnèche-Richard** | [@adebardo](https://github.com/adebardo)   | CS Group                       | [📧](mailto:alice.de-bardonneche-richard@cs-soprasteria.com) |
 
-## Development Lead
+## Emeritus maintainers
 
 ---
 
-- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
-- **Amaury Dehecq** [@adehecq](https://github.com/adehecq)
-- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github.com/erikmannerfelt)
-
+| Emeritus                   | GitHub | Affiliation                | Email                                                        |
+|----------------------------|------|----------------------------|--------------------------------------------------------------|
+| **Erik Schytt Mannerfelt** | [@erikmannerfelt](https://github.com/erikmannerfelt) | University of Oslo         | N/A                                                          |
+| **Emmanuel Dubois**        | [@duboise-cnes](https://github.com/duboise-cnes) | French Space Agency (CNES) | [📧](mailto:emmanuel.dubois@cnes.fr)                         |
 
 ## Contributors
 
 ---
 
+- **Valentin Schaffner** [@vschaffn](https://github.com/vschaffn)
```

Comment:
Is there any logic for the order of contributors? It would make sense to have them ordered by contribution, but might be annoying to keep up-to-date, although one could just revisit it ~annually when updating all other items.

Should we also try to have a minimum requirement for the contributor status? Many of the contributors have 1-2 commits with a few lines added (even @belletva, but we can keep you in the list since you're a maintainer 😉).